mod/discussions/start.php

```diff
@@ -11,6 +11,12 @@
 function discussion_init() {
 
 	elgg_register_page_handler('discussion', 'discussion_page_handler');
+	
+	elgg_register_menu_item('site', [
```

Comment:
all content plugins have their own site menu item... if they have a /all page ... so i think that is consistent